packages/react-charting/src/components/HeatMapChart/__snapshots__/HeatMapChart.test.tsx.snap

```diff
@@ -101,7 +101,7 @@ exports[`HeatMapChart - mouse events Should render callout correctly on mouseove
         transform="translate(40, 0)"
       />
       <g
-        aria-label="Mar/04, p2. Execllent (0-200), 25/2479. Due to unexpected heavy rain, all the pollutants are washed
+        aria-label="Mar/03, p2. Execllent (0-200), 25/2479. Due to unexpected heavy rain, all the pollutants are washed
```

Comment:
Updates in this file don't seem related to the change. Could you confirm.

Reply:
Reverted. This is some artifact of running the unit tests locally.